src/System.Diagnostics.Process/src/System.Diagnostics.Process.csproj

```diff
@@ -402,6 +414,7 @@
     <Reference Condition="'$(TargetGroup)' != 'uap'" Include="Microsoft.Win32.Registry" />
     <Reference Condition="'$(TargetGroup)' == 'uap'" Include="System.Buffers" />
     <Reference Include="System.Collections" />
+    <Reference Include="System.Collections.Concurrent" />
```

Comment:
Can you revert all of the changes to this file except for this one line?

Reply:
Reverted. Sorry for that.